client/profile-wizard/index.js

```diff
@@ -215,6 +221,32 @@ class ProfileWizard extends Component {
 		}
 	}
 
+	skipProfiler() {
+		const { createNotice, updateProfileItems } = this.props;
+		updateProfileItems( {
+			skipped: true,
+			step: this.getCurrentStep().key,
+		} )
+			.then( ( response ) => {
+				if ( response.status === 'success' ) {
+					recordEvent( 'storeprofiler_store_details_skip' );
+					const href = getAdminLink( 'admin.php?page=wc-admin' );
+					getHistory().push( href );
```

Comment:
This one also needs to be updated:

```suggestion
					getHistory().push( getNewPath( {}, '/', {} ) );
```